front/src/modules/activities/timeline/components/Timeline.tsx

```diff
@@ -95,6 +96,17 @@ export function Timeline({ entity }: { entity: CommentableEntity }) {
     },
   });
 
+  const StyledStartIcon = styled.div`
```

Comment:
this Styled code should be put outside (before) of the component to avoid re-renders

Reply:
Sure good catch :)